pubspec.yaml

```diff
@@ -5,8 +5,10 @@ publish_to: none
 
 dependencies:
   auto_route: 5.0.1
+  device_preview: ^1.1.0
   dio: 4.0.6
   elementary: 1.5.0
+  firebase_core_platform_interface: 4.5.1
```

Comment:
?

Reply:
There is an error during building:
```
FirebaseAppPlatform.verifyExtends(_delegate);

                    ^^^^^^^^^^^^^
```
It can be cured with this dependency.
https://github.com/firebase/flutterfire/issues/9806